internal/version/version.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // Repo is the current version of the client libraries in this
 // repo. It should be a date in YYYYMMDD format.
-const Repo = "20200701"
+const Repo = "20200706"
```

Comment:
This seems to be an unrelated change?

Reply:
[Step 7](https://github.com/googleapis/google-cloud-go/blob/master/RELEASING.md#how-to-release-a-submodule). Usually if the regen PR was merged that day, this step makes no difference.